curriculum/challenges/english/25-front-end-development/workshop-binary-search/683ec95f2f8781355556ff7e.md

```diff
@@ -0,0 +1,42 @@
+---
+id: 683ec95f2f8781355556ff7e
+title: Step 6
+challengeType: 20
+dashedName: step-6
+---
+
+# --description--
+
+Now that you've calculated the `mid` index, you need to actually retrieve the value from the `search_list` at that index. This value is what you will compare against your `value` parameter, which is the target you are searching for.
+
+Create a `value_at_middle` variable and assign it the element from `search_list` that is located at the `mid` index. 
+
+# --hints--
+
+You should create a `value_at_middle` variable set to `search_list[mid]`. Don't forget to maintain the same indentation level as the existing variable in the `while` loop.
```

Comment:
I question if this note is needed here about the indentation level.
by this point in the curriculum, campers should understand the importance of indentation in python and be able to resolve indentation errors on their own. 

IMO, it should be removed